### DISCUSSION TITLE
more recipes

### DISCUSSION THREAD

**Discussion by lstebner** [jeremyckahn/farmhand#83] [Ideas]
the game needs more recipes! recipes are a big motivator for which seeds to buy, and how to manage the field. we could also have more cooking related items for sale in the shop such as an oven, and a grill, for things like roasted vegetables. though we could add those recipes early on and require the tool/machine later. 

some ideas: spinach salad, roasted carrots, grilled cheese

**Comment by jeremyckahn** [comment]
I strongly agree! It's not hard to add new recipes, just a little tedious. This commit for adding the Spaghetti recipe can serve as a guide for adding new ones: https://github.com/jeremyckahn/farmhand/commit/39ceca5921a7b65747ba84fa055ea1b3a9f97fc2#diff-5343b242db87de69ccd48c6b6ef9e03651a5c79711aad7e3b516109e53d8f523

I like the idea of implementing specific reusable tools as requirements for recipes. There are only 9 recipes currently, so I think it'd be best to focus on adding more in the vein of what's already there before we start adding in that mechanic. But I am definitely open to it!

**Comment by jmmullins** [comment]
I was going to include this in my comment about cow training / skills, but more applicable here. Yogurt and soap are two other very common products in farm life that are made with milk besides cheese. I saw that some of the recipes get fancy like "spicy pickled garlic", but soap and yogurt seem like easy/natural additions!

Maybe also ice cream and by extension milkshakes? I saw chocolate milk in the recipe list; could have vanilla, chocolate, and rainbow ice cream which make the same in milkshakes by adding more milk

**Comment by jeremyckahn** [comment]
@jmmullins these are great ideas! Rather than making a bunch of yogurt and milkshake variants manually, it might be interesting to explore more of a dynamic crafting system where the player can create completely new types of yogurt and shakes. We've floated this idea [on Discord](https://discord.com/channels/714539345050075176/714539345637408793/1108766901640888360) but haven't fully explored it yet. I think it would cover what your proposing pretty well and keep things interesting!
